app/views/public/api/_method.html.erb

```diff
@@ -5,26 +5,24 @@
       <span><%= method[:path] %></span>
     </p>
     <p><%= method[:description].html_safe %></p>
-    <figure class="code">
-      <pre tabindex="0"><a><%= method[:is_oauth].present? ? "https://gumroad.com#{method[:path]}" : "https://api.gumroad.com/v2#{method[:path]}" %></a></pre>
-    </figure>
+    <%= render "shared/code_snippet" do %>
+<a><%= method[:is_oauth].present? ? "https://gumroad.com#{method[:path]}" : "https://api.gumroad.com/v2#{method[:path]}" %></a>
+    <% end %>
```

Comment:
This is not part of the React migration, so I used a partial for it instead of repeating styles in multiple places